src/js/modules/teamGameplay/actions.js

```diff
@@ -28,7 +29,35 @@ export function requestTeamInfo(teamId) {
       if (err) {
         return dispatch(failTeamInfo(err));
       }
+
       return dispatch(successTeamInfo(data));
     });
   };
 }
+
+export function tryAuthTeamSocket() {
+  return (dispatch, getState) => {
+    const state = getState();
+    const jwt = AuthModule.selectors.getJwt(state);
+
+    authenticateTeam(jwt);
+
+    return dispatch({
+      type: actions.TRY_AUTH_TEAM_SOCKET,
```

Comment:
There's really no reason the auth request should ever fail, so don't fret too much about handling it smoothly.